src/social-web/routes/feed/stage.tsx

```diff
@@ -0,0 +1,145 @@
+/**
+ * Feed Stage
+ *
+ * Main feed list view with DataViews
+ */
+
+import './style.scss';
+import { useMemo, useCallback, useState, useEffect } from '@wordpress/element';
+import { DataViews } from '@wordpress/dataviews';
+import { useView } from '@wordpress/views';
+import type { View, Field } from '@wordpress/dataviews';
+import { __ } from '@wordpress/i18n';
+import { Page } from '../../components/page';
+import { useFeed } from '../../hooks/use-feed';
+import {
+	titleField,
+	dateField,
+	statusField,
+	excerptField,
+	metadataField,
+	createContentField,
+} from '../../components/fields';
+import { getFeedActions } from './FeedActions';
+import type { FeedPost } from '../../types';
+
+const DEFAULT_VIEW: View = {
+	type: 'list',
+	perPage: 20,
+	page: 1,
+	sort: {
+		field: 'date',
+		direction: 'desc',
+	},
+	search: '',
+	filters: [],
+	fields: [ 'metadata', 'title.rendered', 'excerpt.rendered' ],
+	layout: {},
+};
+
+const defaultLayouts = {
+	table: {
```

Comment:
Maybe we don't need to provide the table view for now? It looks pretty broken and I'm not sure about how useful it is?

In the Site Editor, the table view at `/wp-admin/site-editor.php?p=%2Fpage&layout=table` takes over the entire width of the content area, which feels more appropriate. Still, not sure if that is a useful view to read a feed.